go/common/host/host.go

```diff
@@ -1,6 +1,10 @@
 package host
```

Comment:
unrelated to the pr work, but since you're here:
this file should be renamed to `interfaces.go`

Reply:
Yeah I agree tbh. It was called interfaces.go in the /host module but when @joeldudley renamed node to host (which definitely needed doing) it was causing namespace clashes so he put it here to get it out of the package.

I'll look to sort that out in another followup PR because it'll make a mess of this PR to workaround the conflicts Joel discovered.